### PR TITLE
Feature/spotify template

### PR DESCRIPTION
**Pull request by Markuson**

### CODE REVIEW

File: staff/marc-uson/spotify-template/index.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    
+    <header class="header-mbl">
+        <img class="logo" src="images/Spotify_Logo_white.png" alt="logo">
+        <span class="button-mbl"><a href="">VIEW ON SPOTIFY</a></span>
+    </header>
+
+    <aside class="aside">
+        <img class="logo" src="images/Spotify_Logo_white.png" alt="logo">
+        <section class="aside__buttons">
+            <a class="aside__buttons__home" href="">
```
Comment (lexbarba) [inline staff/marc-uson/spotify-template/index.html]: @Markuson BEM naming has to be improved! Once you have used underscore, next children should be added with `-`. Naming would look like: `aside__buttons-home`-

File: staff/marc-uson/spotify-template/style/aspect.sass [hunk 2]
```diff
@@ -0,0 +1,26 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat')
```
Comment (lexbarba) [inline staff/marc-uson/spotify-template/style/aspect.sass:1]: @Markuson nice use of variables! (usually this file is named variables.sass)

File: staff/marc-uson/spotify-template/style/mixin.sass [hunk 3]
```diff
@@ -0,0 +1,23 @@
+@mixin for-galaxy
```
Comment (lexbarba) [inline staff/marc-uson/spotify-template/style/mixin.sass:1]: @Markuson that is the way! Nice use of mixins!

File: staff/marc-uson/spotify-template/index.html (hunk 1 above)
Comment (Markuson) [reply staff/marc-uson/spotify-template/index.html]: OK, done!

File: staff/marc-uson/spotify-template/style/aspect.sass (hunk 2 above)
Comment (Markuson) [reply staff/marc-uson/spotify-template/style/aspect.sass:1]: I called it aspect cause it refers to aspect variables.

maybe better call it "aspect-variables.sass"?

File: staff/marc-uson/spotify-template/style/mixin.sass (hunk 3 above)
Comment (Markuson) [reply staff/marc-uson/spotify-template/style/mixin.sass:1]: thanks! 🤙

File: staff/marc-uson/spotify-template/style/aspect.sass (hunk 2 above)
Comment (lexbarba) [reply staff/marc-uson/spotify-template/style/aspect.sass:1]: as you wish!

**Pull request merged**